src/main/java/io/galeb/manager/entity/Project.java

```diff
@@ -74,4 +74,9 @@ public Project setTeams(Set<Team> teams) {
         }
         return this;
     }
+
+    @Override
+    public String getEnvName() {
+        return null;
```

Comment:
There's a NullPointerException risk. Using generic constant may be a best approach, not?